src/AcceptanceTests/Routing/When_sending_to_specific_namespace.cs

```diff
@@ -126,48 +84,26 @@ public class When_using_multiple_namespaces : NServiceBusAcceptanceTest
         [Test]
         public async Task Should_support_request_reply_across_namespaces_using_contained_endpoint()
         {
-            var runSettings = new RunSettings();
-            runSettings.TestExecutionTimeout = TimeSpan.FromMinutes(1);
-
-            var connectionString = Environment.GetEnvironmentVariable("AzureServiceBusTransport.ConnectionString");
-            var targetConnectionString = Environment.GetEnvironmentVariable("AzureServiceBus.ConnectionString.Fallback");
-
-            var ctx = new AzureServiceBusTransportConfigContext();
-            ctx.Callback = (endpointName, extensions) =>
-            {
-                if (endpointName == Conventions.EndpointNamingConvention(typeof(EndpointInTargetNamespace)))
-                {
-                    extensions.NamespaceRouting().AddNamespace("source", connectionString);
-                    extensions.NamespacePartitioning().AddNamespace("target", targetConnectionString);
-                }
-                else
-                {
-                    extensions.NamespacePartitioning().AddNamespace("source", connectionString);
-                    var targetNamespace = extensions.NamespaceRouting().AddNamespace("target", targetConnectionString);
-                    targetNamespace.RegisteredEndpoints.Add(Conventions.EndpointNamingConvention(typeof(EndpointInTargetNamespace)));
-                }
-
-                var topology = EnvironmentHelper.GetEnvironmentVariable("AzureServiceBusTransport.Topology");
-                if (topology == "ForwardingTopology")
-                {
-                    extensions.UseForwardingTopology();
-                }
-                else
-                {
-                    extensions.UseEndpointOrientedTopology();
-                }
-            };
-
-            runSettings.Set("AzureServiceBus.AcceptanceTests.TransportConfigContext", ctx);
-
-            var context = await Scenario.Define<Context>()
+           var context = await Scenario.Define<Context>()
                 .WithEndpoint<EndpointInSourceNamespace>(b =>
                 {
-                    b.When((bus, c) => bus.Send(new MyRequest()));
+                    b.CustomConfig(c =>
+                    {
+                        var transport = c.ConfigureAzureServiceBus();
+                        transport.NamespacePartitioning().AddNamespace("source", connectionString);
+                        var targetNamespace = transport.NamespaceRouting().AddNamespace("target", targetConnectionString);
+                        targetNamespace.RegisteredEndpoints.Add(Conventions.EndpointNamingConvention(typeof(EndpointInTargetNamespace)));
```

Comment:
I'd still name it `TestingConventions`. `Conventions` is overloaded (Core has it).

Reply:
In the acceptance tests it is almost always the acceptance test conventions. I'm leaving it as is. Much cleaner that way